crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -386,6 +386,17 @@ pub struct StandardMaterial {
     ///
     /// [`Mesh::generate_tangents`]: bevy_render::mesh::Mesh::generate_tangents
     /// [`Mesh::with_generated_tangents`]: bevy_render::mesh::Mesh::with_generated_tangents
+    ///
+    /// # Usage
+    ///
+    /// ```rust
+    /// let normal_handle = asset_server.load_with_settings(
+    ///     "textures/parallax_example/cube_normal.png",
+    ///     // The normal map texture is in linear color space. Lighting won't look correct
+    ///     // if `is_srgb` is `true`, which is the default.
+    ///     |settings: &mut ImageLoaderSettings| settings.is_srgb = false,
+    /// );
+    /// ```
```

Comment:
```suggestion
    /// ```
    /// # use bevy_asset::AssetServer;
    /// # use bevy_ecs::ecs::change_detection::Res;
    /// # use bevy_image::ImageLoaderSettings;
    /// #
    /// fn load_normal_map(asset_server: Res<AssetServer>) {
    ///     let normal_handle = asset_server.load_with_settings(
    ///         "textures/parallax_example/cube_normal.png",
    ///         // The normal map texture is in linear color space. Lighting won't look correct
    ///         // if `is_srgb` is `true`, which is the default.
    ///         |settings: &mut ImageLoaderSettings| settings.is_srgb = false,
    ///     );
    /// }
    /// ```
```

This should fix CI, although it may need to be reformatted :)